deploy/FlapperInit.sol

```diff
@@ -148,29 +133,39 @@ library FlapperInit {
     }
 
     function initSplitter(        
-        DssInstance memory dss,
-        address splitter,
-        SplitterConfig memory cfg
+        DssInstance      memory dss,
+        SplitterInstance memory splitterInstance,
+        SplitterConfig   memory cfg
     ) internal {
+        SplitterLike    splitter = SplitterLike(splitterInstance.splitter);
+        SplitterMomLike mom      = SplitterMomLike(splitterInstance.mom);
+
         // Sanity checks
-        SplitterLike splitter_ = SplitterLike(splitter);
-        require(splitter_.vat()     == address(dss.vat), "Splitter vat mismatch");
-        require(splitter_.daiJoin() == cfg.daiJoin,      "Splitter daiJoin mismatch");
-        require(splitter_.farm()    == cfg.farm,         "Splitter farm mismatch");
+        require(splitter.vat()     == address(dss.vat), "Splitter vat mismatch");
+        require(splitter.daiJoin() == cfg.daiJoin,      "Splitter daiJoin mismatch");
+        require(splitter.farm()    == cfg.farm,         "Splitter farm mismatch");
+
+        require(cfg.hump > 0,         "hump too low");
+        require(cfg.hop >= 5 minutes, "hop too low");
+        require(cfg.burn <= WAD,      "burn too high");
 
-        require(cfg.hop >= 5 minutes, "Splitter hop too low");
-        require(cfg.burn <= WAD,      "Splitter burn too high");
+        splitter.file("hop",  cfg.hop);
+        splitter.file("burn", cfg.burn);
+        splitter.rely(address(mom));
+        splitter.rely(address(dss.vow));
 
-        splitter_.file("hop",  cfg.hop);
-        splitter_.file("burn", cfg.burn);
         FarmLike farm = FarmLike(cfg.farm);
-        farm.setRewardsDistribution(splitter);
-        farm.setRewardsDuration(cfg.rewardsDuration);
+        farm.setRewardsDistribution(splitterInstance.splitter);
+        farm.setRewardsDuration(cfg.hop);
 
-        // Wire splitter with vow
-        splitter_.rely(address(dss.vow));
-        dss.vow.file("flapper", splitter);
+        dss.vow.file("flapper", splitterInstance.splitter);
+        dss.vow.file("hump", cfg.hump);
+        dss.vow.file("bump", cfg.bump);
+
+        mom.setAuthority(dss.chainlog.getAddress("MCD_ADM"));
 
-        dss.chainlog.setAddress(cfg.chainlogKey, splitter);
+        dss.chainlog.setAddress(cfg.chainlogKey, splitterInstance.splitter);
+        dss.chainlog.removeAddress("FLAPPER_MOM");
```

Comment:
Not sure if this way explicit should be part of the init script. Probably I'd follow the same approach of prev chainlog key.

Reply:
And if that is too complex as we already have the splitter key, then I'd say this should be done in the spell separated as it will be a one time thing.

Reply:
Still hard to remember to do so, so yeah not sure what is the best way honestly.

Reply:
Changed to pass all CL keys as bytes32 params.